### PR TITLE
fix(manage): resolve issues around actions buttons in certain scenarios

### PR DESCRIPTION
**Pull request by tomxelliott**
## Describe your changes
Upload Docs - View Representations page changes (CROWN-678)

## Issue ticket number and link
https://pins-ds.atlassian.net/browse/CROWN-948

Screenshot 2025-07-04 at 16 35 15

Screenshot 2025-07-04 at 16 35 30

### CODE REVIEW

File: packages/lib/forms/custom-components/representation-attachments/question.test.js [hunk 1]
```diff
@@ -345,6 +345,39 @@ describe('./lib/forms/custom-components/representation-attachments/question.js',
 				}
 			]);
 		});
+		it('should format action with add only if journey is manage-representations and status is accepted and documents list is empty', () => {
```
Comment (AlexMoulin) [inline packages/lib/forms/custom-components/representation-attachments/question.test.js:348]: It may be a good idea to add a test for when answer is empty to cover that logic?

**Review by AlexMoulin — CHANGES_REQUESTED** [review]
Just one test to add

**Review by AlexMoulin — APPROVED** [review]
